src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -5257,6 +5267,274 @@ def remove_github_actions(cmd, resource_group, name, repo, token=None, slot=None
     return "Disconnected successfully."
 
 
+def add_functionapp_github_actions(cmd, resource_group, name, repo, runtime=None, runtime_version=None, token=None,  # pylint: disable=too-many-statements,too-many-branches
+                                   slot=None, branch='master', build_path=".", login_with_github=False, force=False):
+    if not token and not login_with_github:
+        raise_missing_token_suggestion()
+    elif not token:
+        scopes = ["admin:repo_hook", "repo", "workflow"]
+        token = get_github_access_token(cmd, scopes)
+    elif token and login_with_github:
+        logger.warning("Both token and --login-with-github flag are provided. Will use provided token")
+
+    # Verify resource group, app
+    site_availability = get_site_availability(cmd, name)
```

Comment:
Does this work as expected for cases where the RG & app might be in different Subs? In this case we expect the user to use the resource ID for RG, instead of name.

Reply:
Yes, I checked this case just now